distributed/deploy/tests/test_local_env.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+import sys
+
+import dask
+
+from distributed import Client
+from distributed.core import Status
+from distributed.deploy.local_env import LocalEnvCluster
+
+
+@pytest.mark.asyncio
+async def test_basic():
+    async with LocalEnvCluster(
```

Comment:
Whenever you instantiate `LocalEnvCluster` with `async with` you need to also set the `asynchronous=True` kwarg.